src/SFA.DAS.ASK.Web/Controllers/DeliveryPartner/PlanningMeetings/CheckAnswersController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ASK.Application.Handlers.DeliveryPartner.PlanningMeetings.GetDeliveryPartnerContact;
+using SFA.DAS.ASK.Application.Handlers.DeliveryPartner.PlanningMeetings.GetDeliveryPartnerContacts;
+using SFA.DAS.ASK.Application.Handlers.DeliveryPartner.PlanningMeetings.GetOrCreatePlanningMeeting;
+using SFA.DAS.ASK.Application.Handlers.DeliveryPartner.PlanningMeetings.GetOrganisationContacts;
+using SFA.DAS.ASK.Application.Handlers.RequestSupport.GetOrCreateOrganisationContact;
+using SFA.DAS.ASK.Application.Handlers.RequestSupport.GetSupportRequest;
+using SFA.DAS.ASK.Web.Infrastructure.ModelStateTransfer;
+using SFA.DAS.ASK.Web.ViewModels.DeliveryPartner.PlanningMeetings;
+
+namespace SFA.DAS.ASK.Web.Controllers.DeliveryPartner.PlanningMeetings
+{
+    public class CheckAnswersController : Controller
+    {
+        private readonly IMediator _mediator;
+
+        public CheckAnswersController(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+
+        [HttpGet("delivery-partner/planning-meeting/check-answers/{supportId}")]
+        [ImportModelState]
```

Comment:
Doesn't need the `[ImportModelState]` if this get isn't going to get redirected to after a failed model validation.